File: MVC/Spartane.Core/Domain/Detalle_Laboratorios_Clinicos/Detalle_Laboratorios_Clinicos.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Spartane.Core.Domain.Actividades_del_Evento;
using Spartane.Core.Domain.Indicadores_Laboratorio;

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Web.Mvc;

namespace Spartane.Core.Domain.Detalle_Laboratorios_Clinicos
{
    /// <summary>
    /// Detalle_Laboratorios_Clinicos table
    /// </summary>
    public class Detalle_Laboratorios_Clinicos: BaseEntity
    {
        public int Folio { get; set; }
        public int? Folio_Actividades_del_Evento { get; set; }
        public string Numero_de_Empleado_Titular { get; set; }
        public string Nombre_Completo { get; set; }
        public bool? Familiar_del_Empleado { get; set; }
        public string Numero_de_Empleado_Beneficiario { get; set; }
        public int? Indicador { get; set; }
        public int? Resultado { get; set; }
        public string Unidad { get; set; }
        public string Intervalo_de_referencia { get; set; }
        public DateTime? Fecha_de_Resultado { get; set; }
        public string Estatus_Indicador { get; set; }

        [ForeignKey("Folio_Actividades_del_Evento")]
        public virtual Spartane.Core.Domain.Actividades_del_Evento.Actividades_del_Evento Folio_Actividades_del_Evento_Actividades_del_Evento { get; set; }
        [ForeignKey("Indicador")]
        public virtual Spartane.Core.Domain.Indicadores_Laboratorio.Indicadores_Laboratorio Indicador_Indicadores_Laboratorio { get; set; }

    }
	
	public class Detalle_Laboratorios_Clinicos_Datos_Generales
    {
                public int Folio { get; set; }
        public int? Folio_Actividades_del_Evento { get; set; }
        public string Numero_de_Empleado_Titular { get; set; }
        public string Nombre_Completo { get; set; }
        public bool? Familiar_del_Empleado { get; set; }
        public string Numero_de_Empleado_Beneficiario { get; set; }
        public int? Indicador { get; set; }
        public int? Resultado { get; set; }
        public string Unidad { get; set; }
        public string Intervalo_de_referencia { get; set; }
        public DateTime? Fecha_de_Resultado { get; set; }
        public string Estatus_Indicador { get; set; }

		        [ForeignKey("Folio_Actividades_del_Evento")]
        public virtual Spartane.Core.Domain.Actividades_del_Evento.Actividades_del_Evento Folio_Actividades_del_Evento_Actividades_del_Evento { get; set; }
        [ForeignKey("Indicador")]
        public virtual Spartane.Core.Domain.Indicadores_Laboratorio.Indicadores_Laboratorio Indicador_Indicadores_Laboratorio { get; set; }

    }


}

File: MVC/Spartane.Web/Uploads/Scripts/Detalle_Laboratorios_Clinicos.js
$(document).ready(function () {
    var inpuElementArray = [
        { "inputId": "Folio", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Numero_de_Empleado_Titular", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Nombre_Completo", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Familiar_del_Empleado", "inputType": "checkbox", "IsRequired": false, "IsVisible": true, "IsReadOnly": false, "DefaultValue": null, "HelpText": "" }
        ,{ "inputId": "Numero_de_Empleado_Beneficiario", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Indicador", "inputType": "select", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Resultado", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Unidad", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Intervalo_de_referencia", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Fecha_de_Resultado", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Estatus_Indicador", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }

    ];
    setInputEntityAttributes(inpuElementArray, "#Detalle_Laboratorios_Clinicos", 1);
});


